app/gitcoinbot/README.md

```diff
@@ -27,6 +27,12 @@ To get it running on your repo, you can install it [here](https://github.com/app
 
 Congrats on installing @gitcoinbot.  This bot facilitates gitcoin bounties via github commands.
 
+After Installation you can go ahead <a href="https://github.com/apps/gitcoinbot">here</a> again.
+
+![Gitcoinbot configure](https://github.com/gitcoinco/web/raw/master/docs/imgs/gitcoinbotconfigure.gif)
+
+From there you may select which repositorie you want the GitcoinBot to be working
```

Comment:
some grammar and spelling issues here: 

"After installation you can go here again"

"From there you may select which repository you want the Gitcoinbot to interact with"

Reply:
@molecula451 I think you missed the first bit of my wording change with "After installation you can go here again" - fix that and we're good to go!

Reply:
@danlipert I have fixed please see commits

Reply:
We are now instead using the Congrats on installing Gitcoinbot sentence and then...

"You can go ahead here again" where 'here' contains a direct link to the bot configuration panel